src/Detection/Framework/Astro.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Utopia\Detector\Detection\Framework;
+
+use Utopia\Detector\Detection\Framework;
+
+class Astro extends Framework
+{
+    public function getName(): string
+    {
+        return 'Astro';
```

Comment:
I would recommended lowercase here, so it can match what we store in config/frameworks.php in Appwrite. I would double-check all detection to make sure their getName returns same value as Appwrite can accept in endpoints.